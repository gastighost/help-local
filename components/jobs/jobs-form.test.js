/** @jest-environment jsdom */
import { render } from '@testing-library/react';
import React from 'react';
import JobsForm from './jobs-form';

describe('rendering', () => {
  it('renders job form', () => {
    const { jobsForm } = render(<JobsForm />);
    expect(jobsForm).toMatchSnapshot();
  });

  // it('inputs new job information into the form', () => {

  // });
});

// describe('jobs form', () => {
//   const onSubmit = jest.fn();

//   beforeEach(() => {
//     onSubmit.mockClear();
//     render(<JobsForm onSubmit={onSubmit} />);
//   });

//   it('onSubmit is called when all fields are filled out and pass validation', () => {

//   });
// });
