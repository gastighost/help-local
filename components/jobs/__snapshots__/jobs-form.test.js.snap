// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`insert renders job form 1`] = `undefined`;
